src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorActions/ImageEditorActions.scss

```diff
@@ -0,0 +1,7 @@
+.expansePanel {
+    padding: 24px 24px 32px 0 !important;
```

Comment:
We should probably look at how we can set without using `!important` here. (we can usually override if we use with more specific selectors)

Reply:
those classes are rewritten/renamed by SCSS compiler so you usually put names that make sense for your local component as there will be no side effect
issue is that moonstone define same properties, material ui also so there is already 2 classes with padding. we need !important to be sure it's this one applying

Reply:
Fixed with https://github.com/Jahia/jcontent/pull/536/commits/12ef5999fdfdeec3027d00dd55e061197788190d